src/app/app.cpp

```diff
@@ -74,20 +62,18 @@ static encoder::Encoder create_encoder(Context context, const sc::Monitor& monit
 }
 
 static ExposerPtr create_exposer(const Context& context) {
-  const auto host = context.m_config->get<std::string>("metrics", "127.0.0.1:3228");
+  const auto host = context.m_config->metrics;
   return std::make_unique<prometheus::Exposer>(host);
 }
 
 static NetworkPtr create_network(Context context) {
-  const auto url_str = context.m_config->get<std::string>("url", "tcp://127.0.0.1:1337");
+  const auto url_str = context.m_config->url;
   const auto url = Url::from_string(url_str);
-
-  context.m_logger.info("Streaming to {}", url.to_string());
```

Comment:
Why did you delete this?

Reply:
Because all configuration options are now logged in `Options::read`